setup.py

```diff
@@ -41,5 +43,27 @@ def ensure_version():
         os.remove(version_filename)
 
 
+def generate_django_migrations():
+    from procrastinate.contrib.django.utils import generate_migrations
+
+    generate_migrations()
+
+
+class InstallWrapper(install):
+    def run(self):
+        generate_django_migrations()
+        # Run the standard PyPi copy
+        install.run(self)
+
+
+class DevelopWrapper(develop):
+    def run(self):
+        generate_django_migrations()
+        # Run the standard PyPi copy
+        develop.run(self)
+
+
 with ensure_version() as version:
-    setuptools.setup(version=version)
+    setuptools.setup(
+        version=version, cmdclass={"install": InstallWrapper, "develop": DevelopWrapper}
+    )
```

Comment:
I'm wondering: why not just call generate_django_migrations() at the top level of setup.py ? What are the cases where setup.py runs and we don't want to generate migrations ?